vcap/vcap/device_mapping.py

```diff
@@ -25,7 +25,7 @@ def get_all_devices() -> List[str]:
             #
             # TODO: Use tf.config.list_physical_devices in TF 2.1
 
-            with tf.Session():
+            with tf.compat.v1.Session():
                 all_devices = device_lib.list_local_devices()
```

Comment:
It would be worth confirming that `device_lib.list_local_devices()` still returns device names in the same format as tensorflow v1. 

It's incredibly likely that it still does. I would expect a format like the following:

["CPU:0", "GPU:0", "GPU:1", "XLA stuff", "other irrelevant stuff"]